docs/repositories/pull-requests.md

```diff
@@ -5,11 +5,11 @@ file_name: "pull-requests"
 
 # Quality Pull Requests page
 
-The **Quality Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and the number of new and fixed issues for each pull request. This allows you to monitor the code quality of the work in progress in your repository.
+The **Quality Pull Requests page** displays an overview of the pull requests in your repository, such as the analysis status and the code quality metrics for each pull request. This allows you to monitor the code quality of the work in progress in your repository.
 
-By default, the page lists open pull requests but you can use the drop-down list at the top of the page to display the closed pull requests.
+By default, the page lists open pull requests but you can click the **Closed** tab at the top of the list to display the closed pull requests.
 
-![Pull Requests page](images/pull-requests.png)
+![Pull Requests page](images/pull-requests.png)<!--TODO CY-6556 Update screenshot. It might be tricky to update this screenshot only, because all the remaining screenshots for the Pull Request page are related with one-->
```

Comment:
Let's sync on this during the daily.

Reply:
This is the open source repository that I used to create the original screenshot:

https://app.codacy.com/gh/pyccel/pyccel/pullRequests

You should be able to grab an updated screenshot from that link.